src/RecordProfileForm.js

```diff
@@ -0,0 +1,78 @@
+import React, { useReducer } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import FormSelect from './FormSelect';
+import { db } from './Firebase';
+
+export default () => {
+  const [state, update] = useReducer(
+    (state, update) => ({ ...state, ...update }),
+    {
+      firstName: '',
+      lastName: '',
+    }
+  );
+
```

Comment:
You can load the current document like this:
```
const { user } = useAuthState()
const userDoc = db.collection('users').doc(user.uid)

useEffect(() => {
  const loadUser = async () => {
    const {firstName, lastName} = await userDoc.read();
    update({firstName, lastName});
  }
  loadUser();
}, [])
```

Reply:
Got it. Does this mean I need to overwrite lines 12-15, or to inject this code into the parenthesis, squiggly bracket? 

Not quite sure how this all works together as I'm still learning the structure and syntax here..

Reply:
fixed during tonight's pair coding session

Reply:
fixed during tonight's pair coding session